src/main/java/com/exadel/discount/util/jwt/RefreshJwtUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.exadel.discount.util.jwt;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@Component
+public class RefreshJwtUtil extends JwtUtil{
+    @Value("${jwt.refresh.expiration}")
+    private long REFRESH_TOKEN_EXPIRATION_TIME;
```

Comment:
Why you used one field level injection and for the second one you used static and setter level injection?